src/LodeRunner.API/src/Handlers/RequestLogger/RequestLogger.cs

```diff
@@ -129,16 +134,73 @@ public async Task Invoke(HttpContext context)
             LogRequest(context, cv, ttfb, duration);
         }
 
+        // convert StatusCode for metrics
+        private static string GetPrometheusCode(int statusCode)
+        {
+            if (statusCode >= 500)
+            {
+                return "Error";
+            }
+            else if (statusCode == 429)
+            {
+                return "Retry";
+            }
+            else if (statusCode >= 400)
+            {
+                return "Warn";
+            }
+            else
+            {
+                return "OK";
+            }
+        }
+
+        // get the client IP address from the request / headers
+        private static string GetClientIp(HttpContext context, out string xff)
+        {
+            const string XffHeader = "X-Forwarded-For";
+            const string IpHeader = "X-Client-IP";
+
```

Comment:
Should we move these to a static class for constants that we can use globally?  Header strings seem like something that should be defined for broad usage.

Reply:
I will do it as part of Caching Refactor task